examples/feedback/metrics-server/README.md

```diff
@@ -0,0 +1,350 @@
+# Stateful Model Feedback Metrics Server
```

Comment:
Wonder if adding both `.md` and `.ipynb` file won't lead to these two being out of sync easily.

Reply:
Hmm... the example may be be easier to digest for reader if we'd include some kind of graph at the beginning and short description what's happening

Reply:
I think it's a good point about the `.md` vs `.ipynb` sync. I wonder if it's worth investigating building the `.md` files as part of the build and failing it if there are differences?

Reply:
GitHub renders notebooks quite alright and in documentation sphinx is rendering them through `nblink`s quite well - maybe we can do just fine with only notebook?

Reply:
Yea I think I would go with that.

Reply:
I do agree - I guess the current only advantage is that when people arrive to the example folder they have an easy way to read without having to open up the notebook